### ISSUE TITLE
RBAC needed in advanced reporting

### ISSUE THREAD

**Issue by mmurph3** [kubecost/cost-analyzer-helm-chart#2020]
**What problem are you trying to solve?**
Users are able to see data from other groups (namespace, label, annotations, etc) in the advanced reporting page.

**Describe the solution you'd like**
Would like the same logic available in allocations page applied to advanced reporting, including [labels/annotations 
](https://github.com/kubecost/cost-analyzer-helm-chart/issues/1799)

**Describe alternatives you've considered**
Not providing users access to the UI.

**How would users interact with this feature?**

Users in a certain group would have permissions to only see data/costs for specific groups (ns, labels/annotations) defined in the filters.json and [rbac group ](https://github.com/kubecost/poc-common-configurations/blob/e6c3c433091efc937cc1ea53fb653d8ccd270d5d/saml-okta/values-saml.yaml#L14)

**Comment by teevans** [comment]
Hey Mike, we're working on this right now.

**Comment by teevans** [comment]
Hey @mmurph3 - Advanced reporting uses the same apis as allocations, so in theory here they should operate the same. Do you have a reproduction somewehere?

**Comment by AjayTripathy** [comment]
I can't repro this locally, so let's bump down off of p0. Would more suspect a configuration issue, because this is the intended behavior of advanced reporting.

**Comment by kwombach12** [comment]
@nealormsbee can you help me understand why this is blocked?

**Comment by nealormsbee** [comment]
Hey @kwombach12 I moved this to blocked during the 1.101 testing cycle because of @AjayTripathy comment:

>I can't repro this locally, so let's bump down off of p0. Would more suspect a configuration issue, because this is the intended behavior of advanced reporting.

We needed to be able to reproduce in order to diagnose and fix the issue.

Ultimate resolution was that this was a configuration problem, and so not an urgent bug we needed to address before release. I believe we've discussed adding a view in 103 to let users test their RBAC configurations and diagnose misconfigs.

This should probably be closed as "Cannot Reproduce".
